includes/api/shipping-info.php

```diff
@@ -593,6 +598,54 @@ function smartCodRestriction($addresses, $order)
     return true;
 }
 
+
+function restictPaymentGetway($rzpOrderId){
+
+    // fetch coupon detail from rzp object
+    $razorpay = new WC_Razorpay(false);
+    $api      = $razorpay->getRazorpayApiInstance();
+
+    try
+    {
+        $razorpayData = $api->order->fetch($rzpOrderId);
+
+    } catch (Exception $e) {
+
+        return true;
+    }
+
+    // foreach($razorpayData['promotions'] as $promotion)
```

Comment:
Why this condition is commented?

Reply:
The GC code was not merged at that time, so I commented it , now removed the commented code